### PR TITLE
Updated the code to PHP >= 7.2. Fixed problems with deprecated syntax

### PR DESCRIPTION
**Pull request by Chris2011**
@jkovalsky asked me to host the updater for the download/install statistics, because I asked him sometimes for the numbers. For this I tried the stuff locally and I updated it to PHP7. I changed some old code and I needed to install a php package globally for get it running on PHP >= 7. If this fits I can try to install it on my hetzner server.

### CODE REVIEW

**Comment by Chris2011** [comment]
Jirka and he asked me to host the updater for the download/install statistics, because I asked him sometimes for the numbers. For this I tried the stuff locally and I updated it to PHP7. I changed some old code and I needed to install a php package globally for get it running on PHP >= 7. If this fits I can try to install it on my hetzner server.

**Review by jkovalsky — COMMENTED** [review]
I have provided two minor comments but otherwise it looks good to me. Let's wait what @jpirek has to say.

File: pp3/au/config.php [hunk 1]
```diff
@@ -3,19 +3,18 @@
 /**
  * Config variables 
  */
-DEFINE('LOGFILE_PATH', '/tmp');
+DEFINE('LOGFILE_PATH', './tmp');
 DEFINE('ADMIN_EMAIL', 'jan.pirek@oracle.com');
```
[the remark this answers is not in the document]
Comment (jpirek) [reply pp3/au/config.php:7]: That would be great, Thanks!

**Review by jpirek — APPROVED** [review]

**Comment by matthiasblaesing** [comment]
This is reviewed and bitrot for a year, so lets get it finally in.

**Pull request merged**